spring-integration-jdbc/src/test/java/org/springframework/integration/jdbc/lock/JdbcLockRegistryTests.java

```diff
@@ -502,6 +509,51 @@ void noTableThrowsExceptionOnStart() {
 		}
 	}
 
+	@Test
+	void testUnlock_lockStatusIsExpired_lockHasBeenAcquiredByAnotherProcess_ConcurrentModificationExceptionWillBeThrown() throws Exception {
```

Comment:
We prefer the line length not more than `120` symbols.
Please, consider to shorten the name of this test method.
Plus we prefer a camel case for names.